src/cc/tests/common/StrUtils_T.cc

```diff
@@ -0,0 +1,28 @@
+#include <gtest/gtest.h>
+
+#include "common/StrUtils.h"
+
+namespace KFS {
+namespace Test {
+
+TEST(StrUtils_T, printf) {
```

Comment:
This is a very clear example of how google test is being used by QFS. How about we refer to StrUtils_T.cc as an example in the related section of Developer Documentation?